tools/pip/setup.py

```diff
@@ -103,6 +103,7 @@
                 os.path.join(CURRENT_DIR, 'mxnet/include/nnvm'))
 
 # copy cc file for mxnet extensions
+os.mkdir(os.path.join(CURRENT_DIR, 'mxnet/src'))
 shutil.copy(os.path.join(CURRENT_DIR, 'mxnet-build/src/lib_api.cc'),
             os.path.join(CURRENT_DIR, 'mxnet/src'))
```

Comment:
without creating src folder, the semantics of this is to create a regular file called `src` with the content of lib_api.cc...